packages/openneuro-app/src/scripts/refactor_2021/dataset/mutations/metadata-form.jsx

```diff
@@ -0,0 +1,394 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import TextInput from '../fragments/text-input'
+import SelectInput from '../fragments/select-input'
+import NumberInput from '../fragments/number-input'
+import TextArrayInput from '../fragments/text-array-input'
+import styled from '@emotion/styled'
+
+const Form = styled.form({
+  width: '100%',
+  maxWidth: '700px',
+  margin: '10px 0',
+})
+const InfoText = styled.p({
+  fontWeight: 100,
+  textAlign: 'left',
+})
+const DisabledNote = styled.div({
+  display: 'flex',
+  color: '#5cb85c',
+  alignItems: 'center',
+  i: {
+    marginRight: '0.5rem',
+  },
+  p: {
+    margin: 0,
+  },
+})
+
+const ValidationError = styled.div({
+  display: 'flex',
+  color: 'red',
+
+  i: {
+    marginRight: '0.5rem',
+  },
+})
+
+const metadataFields = hasEdit => {
+  const fields = [
+    {
+      key: 'associatedPaperDOI',
+      label: 'DOI of papers from the source data lab',
+      hoverText:
+        'Papers that were published from the Lab that collected this dataset',
+      component: TextInput,
+      additionalProps: {
+        required: false,
+      },
+    },
+    {
+      key: 'species',
+      label: 'Species',
+      component: SelectInput,
+      additionalProps: {
+        options: [{ value: 'Human' }, { value: 'Rat' }, { value: 'Mouse' }],
+        showOptionOther: true,
+        required: false,
+      },
+    },
+    {
+      key: 'studyLongitudinal',
+      label: 'Study Type',
+      component: SelectInput,
+      additionalProps: {
+        options: [{ value: 'Longitudinal' }, { value: 'Cross-Sectional' }],
+        showOptionOther: true,
+        required: false,
+      },
+    },
+    {
+      key: 'studyDomain',
+      label: 'Domain Studied',
+      component: TextInput,
+      additionalProps: {
+        required: false,
+      },
+    },
+    {
+      key: 'trialCount',
+      label: 'Number of Trials (if applicable)',
+      component: NumberInput,
+      additionalProps: {
+        min: -1,
```

Comment:
Not 0?

Reply:
I just copied this over, not sure why it was -1?